internal/slack/slack.go

```diff
@@ -15,13 +15,16 @@ type Client struct {
 	client        *slack.Client
 	emailMappings map[string]string
 	muteOptions   MuteOptions
+	skip          bool
```

Comment:
This isn't used, right?

Reply:
yeah, a leftover. I extended MuteOptions instead 👍